CITATION.cff

```diff
@@ -0,0 +1,21 @@
+cff-version: 1.2.0
+message: "If you use this software, please cite it as below."
+authors:
+  - family-names: Baker
+    given-names: Cody 
+  - family-names: Sharda
+    given-names: Saksham 
+  - family-names: Mayorquin
+    given-names: Heberto 
+  - family-names: Tauffer
+    given-names: Luiz 
+  - family-names: Buccino
+    given-names: Alessio Paolo 
+    orcid: https://orcid.org/0000-0003-3661-527X
+  - family-names: Dichter 
+    given-names: Benjamin
+    orcid: https://orcid.org/0000-0001-5725-6910
+title: "Nwb-conversion-tools"
+version: 0.9.4
```

Comment:
@alejoe91 Any way we can explicitly link this to the actual current version? Might get annoying to always have to bump both at the same time, and always possible to forget to do so

Reply:
I haven't found anything automatically unfortunately :(

Reply:
```suggestion
version: 0.9.9
```

Bummer; requesting manual bump then

Reply:
I mean, I guess we could have a workflow script or something that checks it as a part of our auto-publish and auto-generates or modifies this file prior to packaging. Not a huge priority though